scripts/perftune.py

```diff
@@ -1211,12 +1291,12 @@ def names():
 
 Default values:
 
- --nic NIC       - default: eth0
+ --nic NIC       - default: eth0 (you can specify multiple nics)
  --cpu-mask MASK - default: all available cores mask
  --tune-clock    - default: false
 ''')
 argp.add_argument('--mode', choices=PerfTunerBase.SupportedModes.names(), help='configuration mode')
-argp.add_argument('--nic', help='network interface name, by default uses \'eth0\'')
+argp.add_argument('--nic', action='append', help='network interface name(s), by default uses \'eth0\'')
```

Comment:
Allowing tuning multiple NICs should be a separate patch

Reply:
it is, am I missing something? you mean like separate PR ?

Reply:
No. Probably I missed it.
Please, rephrase the patch description: the language is confusing. It reads as if you add a support for passing parameters for multiple NICs.

Something like "add a support for tuning multiple NICs in a single command" we be clearer.